src/main/scala/GCD.scala

```diff
@@ -2,15 +2,16 @@
 
 package test
 
-import Chisel._
+import chisel3._
+import chisel3.util._
 
 class GCD extends Module {
   val io = new Bundle {
-    val a  = UInt(INPUT, 32)
-    val b  = UInt(INPUT, 32)
-    val e  = Bool(INPUT)
-    val z  = UInt(OUTPUT, 32)
-    val v  = Bool(OUTPUT)
+    val a  = Input(UInt(32))
+    val b  = Input(UInt(32))
+    val e  = Input(Bool())
+    val z  = Output(UInt(32))
+    val v  = Output(Bool())
```

Comment:
Since the single-argument apply methods are about to be deprecated, we shouldn't use them